examples/README.md

```diff
@@ -8,7 +8,6 @@ Each example is available in both C and in Rust. Examples are accompanied with a
 `*.wat` file which is the wasm input, or a Rust project in a `wasm` folder which
 is the source code for the original wasm file.
 
-Rust examples can be executed with `cargo run --example $name`, and C examples
-need to be compiled using your system compiler and appropriate header files.
+Rust examples can be executed with `cargo run --example $name`. C examples can be built with `mkdir build && cd build && cmake ..`. You can run `cmake --build .` to build all examples or `cmake --build . --target wasmtime-$name`, replacing the name as you wish. They can also be [built manually](https://docs.wasmtime.dev/c-api/).
```

Comment:
A minor style nit: Markdown in this project seems to be pretty consistently word-wrapped around 75-80 columns. Could you word-wrap this paragraph?

Reply:
Oh wow that's really noticeable my bad lol